apps/tables/src/components/ListTables.vue

```diff
@@ -74,7 +74,9 @@ export default {
             )
           );
       } else {
-        return this.schema.tables;
+        return this.schema.tables.filter(
```

Comment:
what about https://github.com/search?q=repo%3Amolgenis%2Fmolgenis-emx2%20schema.tables&type=code  , everywhere the schema.tables is used this filter should be added ?

the endPoint result changes, i woud expect a change in at least one of the api endpoint test that checks for this change

Reply:
for example: ![image](https://github.com/molgenis/molgenis-emx2/assets/47183404/7298e0ff-ea37-46e6-975a-53177d80d4ed)